src/components/Comment/Comments.scss

```diff
@@ -0,0 +1,74 @@
+@import "~styles/main";
+
+.comment {
+  @include font(Poppins);
+  margin: 20px auto;
+  width: 60%;
+
+  //shame: this is very hacky method. If found an alternative, please replace
+  .avatar {
+    position: fixed;
+    margin: 0px -60px;
+  }
+
+  &__top {
+    display: flex;
+    align-items: center;
+    justify-content: space-between;
+    margin: 0 0 5px 0;
+    padding: 0;
+    .comment__author {
+      font-weight: 700;
+      font-size: 0.75rem;
+      display: inline-block;
+      vertical-align: middle;
+      // margin-right: 200px;   // shame- cant give margin right of text
+    }
+    em {
+      font-weight: 400;
+      font-size: 0.66rem;
+      display: block;
```

Comment:
If em is block you does not need to define display block, if em is not block do not make non block element block

Reply:
em is inline element. Not defining block makes is display on same line as author name.

Reply:
than we have Flex for that, But making inline element as block is bad idea